source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+|:--------------------------------------------------------------------- |:------------------------- |:------------------------------- |:--------------------------------------|
+| **Certificate Type**                                                  | Bring your own            | Shared, issued by Let's Encrypt | Bring your own                        |
+| **Renewal**                                                           | Self-managed (up to you)  | Automatic                       | Self-managed (up to you)              |
+| **Inbound IP**                                                        | Static (unique)           | Static (shared)                 | Static (shared)                       |
+| **Client Support**                                                    | 96.02% of browsers        | 95.55% of Browsers <br>Some very old browsers not supported <sup><a href="https://caniuse.com/#search=TLS%201.2">1 <a href="https://caniuse.com/#search=SNI">2</a></sup> | 95.55% of Browsers <br>Some very old browsers not supported <sup><a href="https://caniuse.com/#search=TLS%201.2">1 <a href="https://caniuse.com/#search=SNI">2</a></sup> * |
+| [**SSL Labs Rating**](https://www.ssllabs.com/ssltest/){.external}    | A                         | A+ [with HSTS](/docs/hsts/)     | A+ [with HSTS](/docs/hsts/) *         |
+| **Protocol**                                                          | TLS 1.1 & 1.2             | TLS 1.2 with SNI                | TLS 1.2 with SNI                      |
+| **Ciphers**                                                           | Weak 3DES Cipher          | No Weak 3DES cipher             | No Weak 3DES cipher                   |
+| **Delivery**                                                          | US Datacenter             | [Global CDN](/docs/global-cdn)  | [Global CDN](/docs/global-cdn)        |
+| **Encryption Endpoint**                                               | Load Balancer             | Application Container           | Application Container                 |
+
+\* Browser compatibility and SSL Labs score is guaranteed for shared Let’s Encrypt certificates. The same results are typical for a custom certificate from a mainstream CA with mainstream attributes, but not guaranteed.  For custom certificates, compatibility and SSL Labs score depends on attributes of that certificate, such as number of SAN entries, CA and signing algorithm.
+
+## Frequently Asked Questions
+
+### Do I need a separate certificate for each site in my organization?
+
+Nope! You can use the a single certificate to cover multiple domains spread across various environments or sites. This capability is enabled because the Global CDN uses a technology called Server Name Indication (SNI), which automatically matches inbound requests with an appropriate certificate, including custom certificates.
```

Comment:
Specify "single **wildcard** certificate"

Reply:
a SAN cert can cover multiple domains and may or not include wildcards